File: Java/Addition.java
//Addition program in java.
  
import java.io.*;
class Functions
{
    protected int addNumbers(int a, int b)
    {
    int c= 0;
    c= a+b;
    return c;
    }
}
class Addition extends Functions
{
public static void main(String[] args) throws IOException
{
    int a=0,b=0,c=0;
    BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
    System.out.println("ENter the value of a= ");
    a= Integer.parseInt(br.readLine());
    System.out.println("ENter the value of b= ");
    b= Integer.parseInt(br.readLine());
    Functions obj= new Functions();
    c= obj.addNumbers(a, b);
    System.out.println("Sum of the numbers is= "+c);
}
}

File: Dotnet/Addition.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Addition
{
    class Addition : Functions
    {
        static void Main(string[] args)
        {
            int a1 = 0, b1 = 0, c1 = 0;
            Console.WriteLine("Enter the value of a= ");
            a1 = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine("Enter the value of b= ");
            b1 = Convert.ToInt32(Console.ReadLine());
            Addition obj = new Addition ();
            c1 = obj.addNumbers(a1, b1);
            Console.WriteLine("Sum of the numbers is= " + c1);
            Console.ReadKey();
        }
    }
    public class Functions
    {
        protected int addNumbers(int a, int b)
        {
            int c = 0;
            c = a + b;
            return c;
        }
    }
}
